src/main/java/challenger/mink/users/User.java

```diff
@@ -30,5 +31,12 @@
   private List<Commitment> commitmentList;
   @ManyToOne
   private Challenge challenge;
+  private UserType userType;
```

Comment:
legyen csak role, mert ez a type ez megzavaró részemről, de ha mást nem zavar

Reply:
ne legyen role, mert a Spring Securityben is van "role", és nem lenne jó, ha fogalmilag összekeverednénk